pkg/api/handlers/projects.go

```diff
@@ -717,6 +731,10 @@ func (h *ProjectsHandler) authenticate(req api.Context, local bool) (err error)
 		return err
 	}
 
+	if err = h.OverrideOauthApps(req, &agent, thread); err != nil {
```

Comment:
I would expect that this was needed and this would just be done in the render.go

Reply:
this is moved to render to properly look for parent thread